prep-javascript-functions/main.js

```diff
@@ -0,0 +1,72 @@
+// addTwoNumbers //
+
+function addTwoNumbers(a, b) {
+  return a + b;
+}
+
+var addTwoNumbersResult = addTwoNumbers(2, 2);
+console.log('addTwoNumbers Exercise:', addTwoNumbersResult);
+
+// convertHoursToMinutes //
+
+function convertHoursToMinutes(hours) {
+  return hours * 2;
+}
+
+var convertHoursToMinutesResult = convertHoursToMinutes(60);
+console.log('convertHoursToMinutes Exercise:', convertHoursToMinutesResult);
+
+// getGreeting //
+
+function getGreeting(name) {
+  return name;
+}
+
+var getGreetingResult = getGreeting('"Hello World!"');
+console.log('getGreeting Exercise:', getGreetingResult);
+
+// addAndMultiplyBy5 //
+
+function addAndMultiplyBy5(a, b) {
+  return a + b * 5;
+}
+
+var addAndMultiplyBy5Result = addAndMultiplyBy5(2, 5);
+console.log('addAndMultiplyBy5 Exercise:', addAndMultiplyBy5Result);
+
+// multiplyAndDivideBy5 //
+
+function multiplyAndDivideBy5(a, b) {
+  return a * b / 5;
+}
+
+var multiplyAndDivideBy5Result = multiplyAndDivideBy5(35, 10);
+console.log('multiplyAndDivideBy5 Exercise:', multiplyAndDivideBy5Result);
+
+function subtractTwoNumbers(a, b) {
+  return a - b;
+}
+
+var subtractTwoNumbersResult = subtractTwoNumbers(22, 7);
+console.log('subtractTwoNumbers Exercise:', subtractTwoNumbersResult);
+
+function getCircleCircumference(a) {
+  return a;
+}
```

Comment:
For this function we can multiply `a` by `2` and `Math.PI` to calculate any circle's circumference, given its radius.